client/my-sites/media-library/list-no-content.jsx

```diff
@@ -10,57 +11,65 @@ import EmptyContent from 'components/empty-content';
 import UploadButton from './upload-button';
 import { userCan } from 'lib/site/utils';
 
-export default React.createClass( {
-	displayName: 'MediaLibraryListNoContent',
-
-	propTypes: {
+class MediaLibraryListNoContent extends Component {
+	static propTypes = {
 		site: React.PropTypes.object,
 		filter: React.PropTypes.string,
 		source: React.PropTypes.string,
-	},
+	};
 
 	getLabel() {
-		switch ( this.props.filter ) {
+		const {
+			filter,
+			translate,
+		} = this.props;
+
+		switch ( filter ) {
+			case 'this-post':
+				return translate( 'There are no media uploaded to this post.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?